weave-js/src/components/Panel2/PanelComp.tsx

```diff
@@ -413,14 +413,26 @@ export const RenderTransformerComp = (props: PanelTransformerCompProps) => {
     panelSpec,
     baseConfig
   );
+  const workingChildConfig = useRef(childConfig);
+  workingChildConfig.current = childConfig;
```

Comment:
Haven't dug into this too deeply but this feels a little suspicious to me. I haven't seen this pattern of taking refs on config elsewhere. 

Is there a reason we can't use the `useChildUpdateConfig2` hook defined here? 

https://github.com/wandb/weave/commit/d5fe27daf66f76633bb0bf6d37e24da25aee8a60#diff-3995149dbceb35b00d8f6dfcb37309dcc41cd29347c73bc3bf8eebf01680fdb0R4

Reply:
+1. I think we want to re-use the `useChildUpdateConfig2` helper if possible. (and this extension should probably go into `useSplitTransformerConfig`. `useSplitTransformerConfig` is responsible for creating the `{baseConfig, childConfig, updateChildConfig}` for the child. We should extend it to also support updateConfig2

Reply:
I actually think the best solution is to use the `useUpdateConfig2` hook. We need to operate on the child config, not call a parent updateConfig2 function.

Reply:
I changed to using the hook  -- `updateChildConfig2` is now returned from the `useSplitTransformerConfig` function. This function is used in two places: `RenderTransformerComp` (involved in this drag-and-zoom issue for CombinedPlot) and `ConfigTransformerComp`. Should I use this new `updateChildConfig2` in the `ConfigTransformerComp` as well?